var/spack/repos/builtin/packages/pism/package.py

```diff
@@ -101,7 +106,7 @@ def cmake_args(self):
             ('YES' if '+python' in spec else 'NO'),
             '-DPism_BUILD_ICEBIN=%s' %
             ('YES' if '+icebin' in spec else 'NO'),
-            '-DPism_USE_PROJ4=%s' %
+            '-DPism_USE_PROJ=%s' %
```

Comment:
Was the name of the flag changed? Does this flag work for all versions?

Reply:
The name was changed in 1.2. And the new flag will only work with 1.2. How can I use the old flag for the old version?

Reply:
You can do something like:
```python
args = [
    ...
]

if spec.satisfies('@1.2:'):
    args.append('-DPism_USE_PROJ=%s' %
        ('YES' if '+proj' in spec else 'NO'))
else:
    args.append('-DPism_USE_PROJ4=%s' %
        ('YES' if '+proj' in spec else 'NO'))

return args